lib/util/player.ts

```diff
@@ -158,8 +233,8 @@ export class Player extends AsyncEventEmitter<PlayerEvents> {
         return { TaskID: BigInt(t.ID), TaskCompleted: t.complete };
       }),
     };
-    console.log(thisPlayerData)
-    this.connection?.room.broadcastToAll({
+    console.log(thisPlayerData);
```

Comment:
log

Reply:
Oh yeah this log was already there and I was unsure if i should remove it eheh.

Will do on the next commit

Reply:
> Remove unnecessary logs and it's good, also does it compile (`tsc` not `ts-node`)?

Indeed does not compile since last commit I believe